packaging/windows/clisdk/bundle.wxs

```diff
@@ -45,6 +62,21 @@
   </Bundle>
 
   <Fragment>
+    <util:RegistrySearch
+       Id="VCRedist_KeyExists"
+       Variable="VCRedist_KeyExists"
+       Root="HKLM"
+       Key="SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\$(var.Platform)"
+       Value="Version"
+       Result="exists" />
+    <util:RegistrySearch
+       Id="VCRedist_KeyValue"
+       Variable="VCRedist_KeyValue"
+       Root="HKLM"
+       Key="SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\$(var.Platform)"
+       Value="Version"
+       Result="value" />
```

Comment:
Do you need `Win64` attribute here (since we have an x86 installer too)? 

http://wixtoolset.org/documentation/manual/v3/xsd/wix/registrysearch.html